### ISSUE TITLE
Why the FCN needs a pre-trained weight?

### ISSUE THREAD

**Issue by machanic** [rfuruta/pixelRL#5]
I notice that you use `chainer.serializers.load_npz('../denoise_with_convGRU/model/pretrained_15.npz', net)` to load a pre-trained FCN weight? Why needs the pre-trained FCN weight?
I currently translate the code into PyTorch version, Can I train the FCN directly without loading pretrained weights?
Thank you!

**Comment by rfuruta** [comment]
Thank you for your question.

>Why needs the pre-trained FCN weight?
The pre-trained weights are used in order to make the training of the network stable.

>Can I train the FCN directly without loading?
Maybe yes, but the training will be unstable especially at the beginning.

**Comment by machanic** [comment]
@rfuruta I have an idea of my paper to use your code, but the reward is only a scalar value for the whole image, rather than the pixel-wise reward. Can I use PixelRL? Maybe the difficult is that many agent & action has to determine one reward.
Thank you.

**Comment by rfuruta** [comment]
I do not think PixelRL will work in such a situation. Training PixelRL with one reward is a remaining problem.

**Comment by machanic** [comment]
@rfuruta Why doesn't work? Does it because that so many actions have to be made to obtain one reward?
I have implemented that idea, but it works poorly, maybe my implementation is incorrect?

**Comment by rfuruta** [comment]
I guessed it would not work because the reward value is determined by many actions.
But, my guess may be wrong if your idea is working.

**Comment by dingdingbin** [comment]
@machanic  Hi, can you share us your code of PyTorch version of PixelRL? Thank you very much.

**Comment by Big-OrangeCat** [comment]
> 我注意到您使用`chainer.serializers.load_npz('../denoise_with_convGRU/model/pretrained_15.npz', net)`加载预训练的 FCN 重吗？为什么需要预训练的 FCN 重？ 我现在将代码翻译成 PyTorch 版本，我可以直接训练 FCN 而不是加载预训练的重吗？ 谢谢！
can you share us your code of PyTorch version of PixelRL? Thank you very much.
